src/Sidebar.js

```diff
@@ -25,11 +25,11 @@ const Sidebar = props => {
     <div className="side-bar">
         <div className="side-bar-header">
             <img src={organizationConfig.displayImage} alt="logo"/>
-            <h1>{organizationConfig.displayName}</h1>
+            {/* <h1>{organizationConfig.displayName}</h1> */}
```

Comment:
Minor but maybe better to just remove this code rather than try to comment them out